docs/runtime/runtime-metadata.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "The Metadata of Runtime"
+---
+
+In blockchain, the logic to process transactions and modify storage is usually called STF (State Transition Function). The more popular name among Substrate developers is runtime. 
+
+Substrate module-based metadata provides an overview of blockchain runtime. It helps external code introspect modules, storage items, dispatchable functions and events.
+
+## SRML Metadata
+
+**[Srml-metadata](https://github.com/paritytech/substrate/tree/master/srml/metadata)** is not a "well-known" component of [SRML](runtime/substrate-runtime-module-library.md), since it doesn't provide additional storage entries, dispatchable functions and events. 
+
+It simply provides the data structure to store the overall runtime's metadata, and related codec to convert between raw data and bytes literal. Substrate framework uses the data format called SCALE (Simple Concatenated Aggregate Little-Endian), refer to [Low-level Data Format](overview/low-level-data-format.md) page for more information. 
```

Comment:
"SRML Metadata provides the data structure to store the metadata" --> this is meaningless.

Reply:
When I say srml-metadata is not a well-known component of SRML, I guess users are eager to know what it really is. I do want to improve this expression actually, but I'm not sure if that helps from your perspective.